packages/jss/src/renderers/DomRenderer.js

```diff
@@ -268,7 +289,7 @@ function insertStyle(style: HTMLElement, options: PriorityOptions) {
     return
   }
 
-  getHead().insertBefore(style, prevNode)
```

Comment:
it worked basically before same way, because the insertBefore api allows to pass prevNode === null and it will be inserted in the end of the parent, so the same as appendChild

Reply:
seems a bit simpler to me